src/Renci.SshNet/Channels/Channel.cs

```diff
@@ -183,7 +184,9 @@ public uint RemotePacketSize
             {
                 if (!_remotePacketSize.HasValue)
                 {
+#pragma warning disable S2372 // Exceptions should not be thrown from property getters
                     throw CreateRemoteChannelInfoNotAvailableException();
+#pragma warning restore S2372 // Exceptions should not be thrown from property getters
```

Comment:
Sorry for not pointing it out earlier: there are multiple S2372 suppressions that can be removed (probably a worth a project-wide search)

Reply:
Done.